external/kinesis-asl/src/test/scala/org/apache/spark/streaming/kinesis/KinesisInputDStreamBuilderSuite.scala

```diff
@@ -112,4 +112,38 @@ class KinesisInputDStreamBuilderSuite extends TestSuiteBase with BeforeAndAfterE
     assert(dstream.dynamoDBCreds == Option(customDynamoDBCreds))
     assert(dstream.cloudWatchCreds == Option(customCloudWatchCreds))
   }
+
+  test("should propagate kinesis fetch timestamp values to KinesisInputDStream") {
```

Comment:
Why not just roll this into the previous check? There's a lot of code duplication going on for no other reason than to check that another config value is passed properly.